app/assets/stylesheets/components/_grid.sass

```diff
@@ -8,3 +8,11 @@ $Grid-margin-base: $Theme-spacing-base
   display: inline-block
   margin-left: $Grid-margin-base
   margin-top: $Grid-margin-base
+  vertical-align: top
+
+.Grid-2-columns
```

Comment:
Please point to me where in the SuitCSS docs there's a rule that allows rules written like this.